.travis/2.18-release.yaml

```diff
@@ -0,0 +1,23 @@
+repositories:
+  - name: pulp
+    git_url: https://github.com/pulp/pulp.git
+    git_branch: 2.18-release
+    version: 2.18.0-1
+  - name: pulp_puppet
+    git_url: https://github.com/pulp/pulp_puppet.git
+    git_branch: 2.18-release
+  - name: pulp_rpm
+    git_url: https://github.com/pulp/pulp_rpm.git
+    git_branch: 2.18-release
+  - name: pulp_docker
+    git_url: https://github.com/pulp/pulp_docker.git
+    git_branch: 3.2-release
```

Comment:
3.3-release

Reply:
After irc discussion, we figured it's crane which should be 3.3